### PR TITLE
Feature/open desc

### PR DESCRIPTION
**Pull request by henryju**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "url": "https://jira.sonarsource.com/browse/SLVSCODE",
     "email": "sonarlint@googlegroups.com"
   },
-  "license": "LGPL-3.0",
+  "license": "SEE LICENSE IN LICENSE.txt",
```
Comment (janos-ss) [inline package.json]: I think "See LICENSE.txt" would be enough

Comment (henryju) [inline package.json]: @janos-ss See https://code.visualstudio.com/docs/extensionAPI/extension-manifest

Comment (janos-ss) [inline package.json]: Interesting, good to know!

Comment (henryju) [inline package.json]: Note that this page is specific to vscode, not sure it is standard for all npm modules.

**Pull request merged**